cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -609,6 +611,13 @@ def store_email_info(self, message, status):
         with open(path_email_history_folder + '/email_' + str(number_emails_scratch_dir) + '_' + status + '_' + str(sending_time) +'.email', 'w+') as outfile:
             outfile.write(message.as_string())
 
+    # TODO perhaps move it somewhere else?
+    def get_request_par_dic(self):
+        with open(self.scratch_dir + '/query_output.json') as file:
```

Comment:
this is valid, but there is also analysis_parameters.json with the same data, I suppose. It's also much smaller.